### DISCUSSION TITLE
Why does Demucs (in the Colab, the Anaconda prompt, and the GUI) reduce the volume of stems after they're extracted?

### DISCUSSION THREAD

**Discussion by ghost** [CarlGao4/Demucs-Gui#76] [Q&A]
I tried matching the volume of the split stems and the original sample as close as possible and did phase cancellation tests with all of the extracted stems and the original track. I found that they didn't cancel out completely.
I also found that Demucs reduces the volume of the extracted stems by 1dB. Can anybody in the team fix this?

Note that I was using the htdemucs_ft model.

**Comment by CarlGao4** [comment]
Sorry for late replying. I saw you've found the reason and opened a new issue. I've answered it in #77
